cypress/support/pageBuilder/pbDeleteBlockCategories.js

```diff
@@ -0,0 +1,40 @@
+import { GraphQLClient } from "graphql-request";
+
+Cypress.Commands.add("pbDeleteBlockCategories", () => {
```

Comment:
```suggestion
Cypress.Commands.add("pbAllDeleteBlockCategories", () => {
```

Added `All` since this is what this command does.